### ISSUE TITLE
Browser reload behavior after removing all items from search results

### ISSUE THREAD

**Issue by derekh77** [general-meerkats/dev-dash#21]
If the browser is refreshed after all search results are removed (clicking the "x" on each result) there are no controls displayed and an error is displayed in the console: 

Chrome: 

```
Uncaught TypeError: Cannot read property 'full_name' of undefined
    at populateRepos (repo-select2.js:124)
    at Object.getRepos (repo-select2.js:209)
    at displayMessage (greet.js:134)
    at checkStorage (greet.js:124)
    at Object.init (greet.js:147)
    at HTMLDocument.<anonymous> (app.js:12)
    at j (jquery.min.js:2)
    at k (jquery.min.js:2)

```
Firefox:

`TypeError: repos[0] is undefined[Learn More]  repo-select2.js:124:13
GET 
https://general-meerkats.github.io/dev-dash/dist/css/style.css`

**Comment by belcurv** [comment]
Fixed with merge #24

**Comment by belcurv** [comment]
Fixed with merge #24
